src/content/docs/en/guides/styling.mdx

```diff
@@ -136,14 +136,13 @@ const backgroundColor = "rgb(24 121 78)";
 
 In Astro, HTML attributes like `class` do not automatically pass through to child components.
 
-
-Instead, accept a `class` prop in the child component and apply it to the root element. When destructuring, you must rename it, because `class` is a [reserved word](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Lexical_grammar#reserved_words) in JavaScript.
+Instead, accept a `class` prop in the child component and apply it to the root element. When destructuring, you must rename it, because `class` is a [reserved word](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Lexical_grammar#reserved_words) in JavaScript. The default styles scoping strategy of Astro also requires you to pass the `data-astro-cid-*` attribute, and the simplest way to do this is by taking the `...rest` of the props and forwarding it to the component.
```

Comment:
```suggestion
Instead, accept a `class` prop in the child component and apply it to the root element. When destructuring, you must rename it, because `class` is a [reserved word](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Lexical_grammar#reserved_words) in JavaScript. 

When using the default `scopedStyleStrategy: 'attribute'`, you must also pass the `data-astro-cid-*` attribute. You can do this by also passing the `...rest` of the props to the component.
```
Just making some shorter, more active sentences!

Reply:
Right, I see what the goal is.

After looking at the suggested wording, I wrote a response and deleted it because I have not noticed the word "default". Is this also an indication of it not standing out enough? What do you think about this alternative?

> You must also pass the `data-astro-cid-*` attribute, unless you have changed `scopedStyleStrategy` to `class` or `where`. You can do this by passing the `...rest` of the props to the component.

Reply:
Ooh, that makes a lot of sense, yes! Even better than the original, I think!

Reply:
Thank you! I have updated the PR to use this sentence, could you please review it again?